container-host-files/etc/scf/config/opinions-common.yml

```diff
@@ -357,8 +356,7 @@ properties:
     service_name: 'persi-nfs'
     service_id: '20e22be2-a4fb-4b17-a5ef-0c9dba8bdc74'
   router:
-    backends:
-      enable_tls: true
+    ca_certs: ''
```

Comment:
This should be customizable (or at least stick a Trello card in)

Reply:
IMHO trello card is good enough for now.

Reply:
Ref: https://trello.com/c/2ZUa6mv4/625-make-routercacerts-customizable